.github/workflows/examples.yml

```diff
@@ -119,8 +116,9 @@ jobs:
         cd example_js/react18_ssr
         npm install
         npm run build
+        cd ../..
         echo '> start wasmedge'
-        nohup wasmedge --dir .:. --dir ./modules:../../modules ../../target/wasm32-wasi/release/wasmedge_quickjs.wasm dist/main.mjs &
+        nohup wasmedge --dir .:. target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/react18_ssr/dist/main.mjs &
```

Comment:
`wasmedge --dir .:. --dir ./public:./example_js/react18_ssr/public target/wasm32-wasi/release/wasmedge_quickjs.wasm example_js/react18_ssr/dist/main.mjs`

this `main.js` will load `public/main.css`

Reply:
Hmm, do you mean that we should always start the SSR server from its own directory?

Reply:
Ok. I think I understand it now.